tasks/main.yml

```diff
@@ -45,16 +45,17 @@
       - docker-ce
       - docker-ce-cli
       - containerd.io
+  when: not ansible_check_mode
```

Comment:
why are you using `not ansible_check_mode` here?

the [ansible docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html#attributes) say: 

> Can run in check_mode and return changed status prediction without modifying target, if not supported the action will be skipped.